src/components/common/base-card/base-card.styles.tsx

```diff
@@ -0,0 +1,37 @@
+import { Color } from "@lifesg/react-design-system/color";
+import styled, { css } from "styled-components";
+
+// =============================================================================
+// STYLE INTERFACES
+// =============================================================================
+interface IBodyStyleProps {
+    $focused: boolean;
+}
+
+// =============================================================================
+// STYLING
+// =============================================================================
+export const Body = styled.div<IBodyStyleProps>`
+    padding: 1rem;
+    cursor: pointer;
+    border-radius: 0.5rem;
+    width: 100%;
+    position: relative;
+    display: flex;
+    align-items: center;
+    box-shadow: 0px 2px 8px 0px rgba(104, 104, 104, 0.24);
+
+    ${({ $focused }) =>
+        $focused
+            ? css`
+                  border: 1px solid ${Color.Primary};
+                  background: ${Color.Accent.Light[5]};
+              `
+            : css`
+                  border: 1px solid transparent;
+              `}
+
+    :hover {
+        border-color: ${Color.Primary};
+    }
```

Comment:
understand we are deviating from figma by not applying a different box-shadow colour, can we have a TODO comment to indicate that? this affects both hover and hover in focused states